docs/quickstart.md

```diff
@@ -1,246 +1,67 @@
-    "lastModifiedOn": 1572381585,
-    "leaseStatus": "Active",
-    "leaseStatusModifiedOn": 1572381585,
-    "leaseStatusReason": "",
-    "principalId": "DCEPrincipal"
-}
-```
-
-After getting the response, call the *${api_url}/accounts* endpoint 
-again to see that the account status has been changed to 
-`Leased`:
-
-```json
-[
-    {
-        "accountStatus": "Leased",
-        "adminRoleArn": "arn:aws:iam::123456789012:role/DCEAdmin",
-        "createdOn": 1572379783,
-        "id": "123456789012",
-        "lastModifiedOn": 1572381585,
-        "metadata": null,
-        "principalPolicyHash": "\"852ee9abbf1220a111c435a8c0e65490\"",
-        "principalRoleArn": "arn:aws:iam::123456789012:role/DCEPrincipal"
-    }
-]
-```
-Once you see the first lease provisioned in the system, you are ready to use your
-first lease! See [logging into your leased account](howto.md#login-to-your-dce-account).
-
-### Listing leases
-
-To list the leases, use an HTTP GET request to the *${api_url}/leases* endpoint
-to see the response as shown here:
-
-```json
-[
-    {
-        "accountId": "123456789012",
-        "budgetAmount": 20,
-        "budgetCurrency": "USD",
-        "budgetNotificationEmails": [
-            "myuser@example.com"
-        ],
-        "createdOn": 1572381585,
-        "expiresOn": 1572382800,
-        "id": "94503268-426b-4892-9b53-3c73ab38aeff",
-        "lastModifiedOn": 1572381585,
-        "leaseStatus": "Active",
-        "leaseStatusModifiedOn": 1572381585,
-        "leaseStatusReason": "Active",
-        "principalId": "DCEPrincipal"
-    }
-]
-```
-
-### Destroying leases
-
-Leases can automatically expire based on a date or a budget amount, but
-leases may also be administratively destroyed at any time. To destroy
-a lease with the API, send a DELETE request to *${api_url}/leases
-with the following request body:
-
-```json
-{
-    "principalId": "DCEPrincipal",
-    "accountId": "123456789012"
-}
-```
-
-The API response for a successful lease destroy looks like this:
-
-```json
-{
-    "accountId": "519777115644",
-    "budgetAmount": 20,
-    "budgetCurrency": "USD",
-    "budgetNotificationEmails": [
-        "john.doe@example.com"
-    ],
-    "createdOn": 1572381585,
-    "expiresOn": 1572382800,
-    "id": "94503268-426b-4892-9b53-3c73ab38aeff",
-    "lastModifiedOn": 1572442028,
-    "leaseStatus": "Inactive",
-    "leaseStatusModifiedOn": 1572442028,
-    "leaseStatusReason": "Destroyed",
-    "principalId": "jdoe123"
-}
-```
+Deploy DCE and lease an account quickly using the DCE CLI.
+
+1. Download the appropriate executable for your OS from the [latest release](https://github.com/Optum/dce-cli/releases/latest). e.g. for mac, you should download dce_darwin_amd64.zip
+
+1. Unzip and move the executable to a directory on your PATH, e.g.
+
+    ```
+    # Download the zip file
+    wget https://github.com/Optum/dce-cli/releases/download/<VERSION>/dce_darwin_amd64.zip
```

Comment:
I think you can just replace `<VERSION>` with `latest` here, so this is a working example